test/e2e/bindata/assets/05_route.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: openshift-cli-manager
+  namespace: openshift-cli-manager-operator
+  annotations:
+    haproxy.router.openshift.io/timeout: 5m
+  labels:
+    app: openshift-cli-manager
+spec:
+  path: /cli-manager
+  to:
+    kind: Service
+    name: openshift-cli-manager
+    weight: 100
+  port:
+    targetPort: cli-manager-port
+  tls:
+    termination: edge
+    insecureEdgeTerminationPolicy: Allow
```

Comment:
In production, this value is `Redirect` to redirect http requests to https. However, in e2e tests, in order to serve binaries in http, we have to allow http usage.

Reply:
What would it take to serve the binaries in https?

Reply:
We need to add self-signed CA bundle as trusted in the CI host to `http.Get` simply working. However, the user we have is very limited access and even can't simply access directories in `/etc/...`